alert/store.go

```diff
@@ -527,6 +529,14 @@ func (s *Store) _create(ctx context.Context, tx *sql.Tx, a Alert) (*Alert, *aler
 		return nil, nil, err
 	}
 
+	// insert alert metadata
+	err = s.insertMetaData(tx, ctx, a.ID, a.Meta.AlertMetaV1)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	ctx = log.WithFields(ctx, log.Fields{"AlertID": a.ID, "Meta": a.Meta.AlertMetaV1})
+	log.Logf(ctx, "Alert created.")
```

Comment:
I think we already have log entries for this, also, I don't think we want to log metadata since it's specific to the client and potentially large